spec/features/user_visits_login_page_spec.rb

```diff
@@ -2,12 +2,12 @@
 
 RSpec.feature "User visits login page" do
   context "they can't log in" do
-    it "with a bad email" do
-      user = User.create(username: "", password: "pass")
+    it "with a bad username" do
+      user = User.create(username: "John Elway", password: "je", password_confirmation: "je", email: 'je@je.com', cellphone: '1234561234')
 
```

Comment:
you need to create a valid user here, otherwise the expectations are just passing because the user doesn't exist.

for this login stuff i had to manually create the user for the bcrypt hash to be valid